data/forms/adult-referral.jsx

```diff
@@ -173,12 +173,6 @@ export default {
             'Community/Other route',
           ],
         },
-        {
-          component: 'Radios',
-          name: 'presentingIssueGen',
-          label: 'Presenting Issue',
-          options: ['Female', 'Male', 'Unknown', 'Other'],
-        },
```

Comment:
Don't we need this? 🤔

Reply:
No this is a copy and paste error from gender.